### PR TITLE
chore: polish v0.4 upgrade instructions

### PR DESCRIPTION
**Pull request by dpc**

### CODE REVIEW

**Review by bradleystachurski — APPROVED** [review]
Good improvements 👍

File: docs/RELEASE_NOTES-v0.4.md [hunk 1]
```diff
@@ -6,16 +6,20 @@ For all the non-critical release notes, see usual places (like github
...
+binaries.
+
+If you are setting up a new Federation using 0.4.x version, you don't need to do anything
+special, but you do need to acknowledge being aware of this requirement (see last section
+of this document).
 
-All guardians should be available to coordinate upgrading using an out-of-band communicaitons
-channel, e.g. a group chat. Once all guardians are available, get the latest session count so
-a session in the future can be chosen to coordinate a shutdown.
+**You must first upgrade your 0.3.x Federation to version v0.3.3 or higher for functionality
```
Comment (bradleystachurski) [inline docs/RELEASE_NOTES-v0.4.md:18]: [nit] Existing federations can still upgrade from 0.2.1 or 0.2.2, so it might be confusing to read `0.3.x`.

Comment (bradleystachurski) [reply docs/RELEASE_NOTES-v0.4.md:18]: I'm going to merge and backport to releases/v0.4. If you want to update, please feel free to open another PR

**Comment by fedimint-backports** [comment]
Successfully created backport PR for `releases/v0.4`:
- #5681

**Pull request merged**